content/en/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions.md

```diff
@@ -347,6 +347,63 @@ kubectl create -f my-crontab.yaml
 crontab "my-new-cron-object" created
 ```
 
+### Publish Validation Schema in OpenAPI v2
+
+{{< feature-state state="alpha" for_kubernetes_version="1.14" >}}
+
+Starting with Kubernetes 1.14, [custom resource validation schema](#validation) can be published as part
+of [OpenAPI v2 spec](docs/concepts/overview/kubernetes-api.md#openapi-and-swagger-definitions) from
```

Comment:
Delete .md in link --> /docs/concepts/overview/kubernetes-api/#openapi-and-swagger-definitions